docs/UserGuide.md

```diff
@@ -615,7 +617,7 @@ To delete everyone at the same time, please refer to the <a href="#clearing-all-
 
 #### Locating persons by name or contact number : `find`
 
-Finds persons whose names match any of the given keywords, or phone numbers contain any of the given keywords (in digits).
+Finds persons whose names match any of the given keywords, or phone numbers start with any of the given keywords (in digits).
```

Comment:
I don't know about you but this sentence is grammatically correct but feels awkward?
Would
> Finds the names of persons that match the given keywords, or phone numbers of persons that start with any of the given keywords.
be better?

Reply:
Updated to

> Finds persons whose names match any of the given keywords, or persons whose phone numbers start with any of the given keywords (in digits).